charts/victoria-logs-single/values.yaml

```diff
@@ -307,15 +307,16 @@ vector:
   role: Agent
   dataDir: /vector-data-dir
   resources: {}
+  podMonitor:
+    enabled: true
```

Comment:
I would suggest keeping it disabled by default as it requires an external CRD to be present in cluster.

Reply:
updated